### ISSUE TITLE
docker Jira Endless Loop: StoryDataClientImpl 

### ISSUE THREAD

**Issue by DimsumPanda** [capitalone/Hygieia#1895]
Before I added the JIRA_PAGE_SIZE=10, I was gettting a socket timeout error. However, when I added this environment variable, it goes in an endless loop through pages 10 by 10 and the data is still not populated in the dashboard.

```
hygieia-jira     | 2017-12-08 16:11:00,012 INFO  c.c.d.collector.CollectorTask - ------------------------------- ----
hygieia-jira     | 2017-12-08 16:11:00,013 INFO  c.c.d.collector.CollectorTask - https://{mycompany}/jira
hygieia-jira      | 2017-12-08 16:11:00,013 INFO  c.c.d.collector.CollectorTask - ------------------------------- ----
hygieia-jira      | 2017-12-08 16:11:00,027 INFO  c.c.d.collector.CollectorTask - Team Data               0         0s
hygieia-jira      | 2017-12-08 16:11:01,739 INFO  c.c.d.collector.CollectorTask - Project Data           62         1s
hygieia-jira      | 2017-12-08 16:11:06,617 INFO  c.c.d.c.story.StoryDataClientImpl - Loop i 0 pageSize 10
hygieia-jira      | 2017-12-08 16:11:10,339 INFO  c.c.d.c.story.StoryDataClientImpl - Loop i 10 pageSize 10

```
My docker-compose:
```
hygieia-jira-feature-collector:
  image: hygieia-jira-feature-collector:latest
  container_name: hygieia-jira
  links:
  - mongodb
  - hygieia-api
  environment:
  - JIRA_BASE_URL=https://{mycompany}.com/jira
  - JIRA_CREDENTIALS={base64Credentials}
  - JIRA_ISSUE_TYPE_NAMES=Story
  - MONGO_PORT=tcp://mongodb:27017
  - JIRA_PAGE_SIZE=10
```
![jiraissue](https://user-images.githubusercontent.com/10524720/33778722-f71c3552-dc17-11e7-9a6d-f472819f97c4.JPG)

**Comment by subodhbattina** [comment]
Hi @DimsumPanda,
Could you please try again with all these properties below and let us know if it works for you? Thanks !

# PageSize - Expand contract this value depending on Jira implementation's
# default server timeout setting (You will likely receive a SocketTimeoutException)
pageSize=100
queryFolder=jiraapi-queries
storyQuery=story
epicQuery=epic

# Add these if you are using a proxy
jiraProxyUrl=<Full Domain Name of the Proxy>
jiraProxyPort=<Proxy Port>

jiraBaseUrl=<Base URL for Jira>
jiraQueryEndpoint=rest/api/2/

# For basic authentication, requires username:password as string in base64
# This command will make this for you:  echo -n username:password | base64
jiraCredentials=<base64 credentials>

jiraIssueTypeNames=Story

#############################################################################
# In Jira, your instance will have its own custom field created for 'sprint' or 'timebox' details,
# which includes a list of information.  This field allows you to specify that data field for your
# instance of Jira. Note: You can retrieve your instance's sprint data field name
# via the following URI, and look for a package name com.atlassian.greenhopper.service.sprint.Sprint;
# your custom field name describes the values in this field:
# https://[your-jira-domain-name]/rest/api/2/issue/[some-issue-name]
#############################################################################
jiraSprintDataFieldName=customfield_10007

#############################################################################
# In Jira, your instance will have its own custom field created for 'super story' or 'epic' back-end ID,
# which includes a list of information.  This field allows you to specify that data field for your instance
# of Jira.  Note:  You can retrieve your instance's epic ID field name via the following URI where your
# queried user story issue has a super issue (for example, epic) tied to it; your custom field name describes the
# epic value you expect to see, and is the only field that does this for a given issue:
# https://[your-jira-domain-name]/rest/api/2/issue/[some-issue-name]
#############################################################################
 jiraEpicIdFieldName=customfield_10003

#############################################################################
# In Jira, your instance will have its own custom field created for 'story points'
# This field allows you to specify that data field for your instance
# of Jira.  Note:  You can retrieve your instance's storypoints ID field name via the following URI where your
# queried user story issue has story points set on it; your custom field name describes the
# story points value you expect to see:
# https://[your-jira-domain-name]/rest/api/2/issue/[some-issue-name]
#############################################################################
 jiraStoryPointsFieldName=customfield_10004

#############################################################################
# In Jira, your instance will have its own custom field created for 'team'
# This field allows you to specify that data field for your instance
# of Jira.  Note:  You can retrieve your instance's team ID field name via the following URI where your
# queried user story issue has team set on it; your custom field name describes the
# team value you expect to see:
# https://[your-jira-domain-name]/rest/api/2/issue/[some-issue-name]
#############################################################################
 jiraTeamFieldName=customfield_11248 
# Set this to true if you use boards as team
 jiraBoardAsTeam=true

Thanks !

**Comment by DimsumPanda** [comment]
Sorry I haven't updated - I actually ran into other issues, but haven't been working on the dashboard for a bit. 
I don't quite recall what I fixed, but the changes since I posted include:
- Choosing "Any" for the Team Name
- Removing logs from my hygieia-api container (I didn't have storage mounted on my test AWS instance)
- For some reason the Jira stories were listed under "Kanban" instead of "Scrum". I think this change was what resolved it for me.
